gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-2-entrypoints/step-2-entrypoints-2-config.component.ts

```diff
@@ -30,6 +32,10 @@ export class Step2Entrypoints2ConfigComponent implements OnInit, OnDestroy {
   private unsubscribe$: Subject<void> = new Subject<void>();
 
   public formGroup: FormGroup;
+  public selectedEntrypoints: { id: string; name: string }[];
+  public entrypointSchemas: Record<string, GioJsonSchema>;
+  public entrypointInitialValues: Record<string, any>;
+  public entrypointFormGroups: Record<string, FormGroup>;
```

Comment:
WDYT to try to remove this by `formGroup.get(entrypoint.id)` directly in template ?

Reply:
and try to have something like this 
```
this.formGroup = currentStepPayload.selectedEntrypoints.reduce...
```
🤷‍♂️ ?